helm-charts/splunk-otel-collector/templates/configmap-fluentd.yaml

```diff
@@ -288,91 +286,24 @@ data:
-        {{- end }}
-        {{- with .Values.logsBackend.splunkIngestAPI.ingestAPIHost }}
-        ingest_api_host {{ . }}
-        {{- end }}
-        {{- with .Values.logsBackend.splunkIngestAPI.tenant }}
-        ingest_api_tenant {{ . }}
-        {{- end }}
-        {{- with .Values.logsBackend.splunkIngestAPI.eventsEndpoint }}
-        ingest_api_events_endpoint {{ . }}
-        {{- end }}
-        {{- with .Values.logsBackend.splunkIngestAPI.debugIngestAPI }}
-        debug_http {{ . }}
-        {{- end }}
-        {{- else }}
-        @type splunk_hec
-        {{- with .Values.logsBackend.hec.protocol | default .Values.ingestProtocol }}
-        protocol {{ . }}
-        {{- end }}
-        {{- with .Values.logsBackend.hec.host | default (include "splunk-otel-collector.ingestHost" .) }}
-        hec_host {{ . | quote }}
-        {{- end }}
-        {{- with .Values.logsBackend.hec.port | default .Values.ingestPort }}
-        hec_port {{ . }}
-        {{- end }}
-        hec_token "#{ENV['SPLUNK_HEC_TOKEN']}"
-        index_key index
-        insecure_ssl {{ .Values.logsBackend.hec.insecureSSL | default false }}
-        {{- if .Values.logsBackend.hec.clientCert }}
-        client_cert /fluentd/etc/splunk/hec_client_cert
-        {{- end }}
-        {{- if .Values.logsBackend.hec.clientKey }}
-        client_key /fluentd/etc/splunk/hec_client_key
-        {{- end }}
-        {{- if .Values.logsBackend.hec.caFile }}
-        ca_file /fluentd/etc/splunk/hec_ca_file
-        {{- end }}
-        {{- end }}
-        host "#{ENV['K8S_NODE_NAME']}"
-        source_key source
-        sourcetype_key sourcetype
-        <fields>
-          # currently CRI does not produce log paths with all the necessary
-          # metadata to parse out pod, namespace, container_name, container_id.
-          # this may be resolved in the future by this issue: https://github.com/kubernetes/kubernetes/issues/58638#issuecomment-385126031
-          {{- if eq .Values.fluentd.config.containers.logFormatType "cri"}}
-          container_retry
-          {{- else }}
-          container.image.name container_image
-          {{- end }}
-          k8s.pod.uid pod_uid
-          k8s.pod.name pod
-          k8s.container.name container_name
-          k8s.namespace.name namespace
-          k8s.node.name node_name
-          k8s.cluster.name cluster_name
-          container.id container_id
-          host.name
-          {{- range .Values.extraAttributes.custom }}
-          {{ .name }}
-          {{- end }}
-          {{- range .Values.indexFields }}
-          {{ . }}
-          {{- end }}
-          {{- range .Values.extraAttributes.podLabels }}
-          k8s.pod.labels.{{ . }}
-          {{- end }}
-          {{- if .Values.environment }}
-          deployment.environment
-          {{- end }}
-        </fields>
+        @type forward
+        heartbeat_type udp
+        <server>
+          host 127.0.0.1
+          port 8006
+        </server>
```

Comment:
Why do we need this one?

Reply:
It's the destination of the logs (the agent).

Reply:
Oh right!